settings.php

```diff
@@ -224,81 +235,87 @@
- * malicious client could bypass restrictions by setting the
- * X-Forwarded-For header directly. Therefore, Backdrop's proxy
- * configuration requires the IP addresses of all remote proxies to be
- * specified in $settings['reverse_proxy_addresses'] to work correctly.
- *
- * Enable this setting to get Backdrop to determine the client IP from
- * the X-Forwarded-For header (or $settings['reverse_proxy_header'] if set).
- * If you are unsure about this setting, do not have a reverse proxy,
- * or Backdrop operates in a shared hosting environment, this setting
- * should remain commented out.
- *
- * In order for this setting to be used you must specify every possible
- * reverse proxy IP address in $settings['reverse_proxy_addresses'].
- * If a complete list of reverse proxies is not available in your
- * environment (for example, if you use a CDN) you may set the
- * $_SERVER['REMOTE_ADDR'] variable directly in settings.php.
- * Be aware, however, that it is likely that this would allow IP
+ * Reverse Proxy Configuration.
+ *
+ * Reverse proxy servers are often used to enhance the performance of heavily
+ * visited sites and may also provide other site caching, security, or
+ * encryption benefits. In an environment where Backdrop is behind a reverse
+ * proxy, the real IP address of the client should be determined such that the
+ * correct client IP address is available to Backdrop's logging and access
+ * management systems. In the most simple scenario, the proxy server will add an
+ * X-Forwarded-For header to the request that contains the client IP address.
+ * However, HTTP headers are vulnerable to spoofing, where a malicious client
+ * could bypass restrictions by setting the X-Forwarded-For header directly.
+ * Therefore, Backdrop's proxy configuration requires the IP addresses of all
+ * remote proxies to be specified in $settings['reverse_proxy_addresses'] to
+ * work correctly.
+ *
+ * Enable this setting to get Backdrop to determine the client IP from the
+ * X-Forwarded-For header (or $settings['reverse_proxy_header'] if set). If you
+ * are unsure about this setting, do not have a reverse proxy, or Backdrop
+ * operates in a shared hosting environment, this setting should remain
+ * commented out.
+ *
+ * In order for this setting to be used, you must specify every possible reverse
+ * proxy IP address in $settings['reverse_proxy_addresses']. If a complete list
+ * of reverse proxies is not available in your environment (for example, if you
+ * use a CDN), then you may set the $_SERVER['REMOTE_ADDR'] variable directly in
+ * settings.php. Be aware, however, that it is likely that this would allow IP
  * address spoofing unless more advanced precautions are taken.
  */
 // $settings['reverse_proxy'] = TRUE;
 
 /**
- * Specify every reverse proxy IP address in your environment.
- * This setting is required if $settings['reverse_proxy'] is TRUE.
+ * Reverse proxy IP address list.
```

Comment:
If the single words are capitalized, that should be consistently done.
The previous section is *Reverse Proxy Configuration.* but the one before that is *Maintenance theme.*